mod.ts

```diff
@@ -92,6 +92,14 @@ export class Udd {
       url.url = `${url.at(initVersion.slice(1)).url}#${newFragmentToken}`;
     }
 
+    try {
+      new Semver(url.version());
+    } catch (e) {
+      // probably, the version string is a branch name.
+      await this.progress.log(`Using a branch: ${url.url}`);
```

Comment:
Is `url.version()` the branch in this case (we expect)? I think worth saying branch name too.
i.e. `Using a branch (${url.version()}): ${url.url}`  ?

I worry it can throw for some other reason... perhaps it should be:
```
const version = url.version()
try {...
```
so we are only catching the Semver constructor (which I assume is what throws here?)

Reply:
> I worry it can throw for some other reason... perhaps it should be:

I think we don't need to modify it.  Because it seems to be assumed that `url.version()` never fails.  See:
https://github.com/hayd/deno-udd/blob/master/mod.ts#L80

Reply:
I updated the log message and the comment in b97aecaab7aa8743f9ac502a17c625758067ce55.

IMO, we don't need to show `url.version()` in the log message.  Because it's included in `url.url` if developers specify the version in the URL, and they know what they do.